doc/modules/classes.rst

```diff
@@ -143,43 +143,58 @@ Classes
    covariance.oas
    covariance.graph_lasso
 
+.. _model_selection_ref:
```

Comment:
As I said, I think I would keep the modules in the references under a "old and will be removed" header. I'm not sure what @vladn @gaelvaroquaux or @jnothman think about that, though.